@here/harp-materials/lib/ShaderChunks/ShadowChunks.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (C) 2017-2019 HERE Europe B.V.
+ * Licensed under Apache 2.0, see full license in LICENSE
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export const correctShadowChunk = `
+    struct PhysicalMaterial {
+        vec3	diffuseColor;
+        float	specularRoughness;
+        vec3	specularColor;
+    };
+
+    #define DEFAULT_SPECULAR_COEFFICIENT 0.04
```

Comment:
Is this define still needed with the new dummy specular light calculation?

Reply:
Yes, it is necessary in another shader chunk from threejs, I tried to remove, but was unsuccessful.